taglib/toolkit/tfile.h

```diff
@@ -66,17 +66,17 @@ namespace TagLib {
      * Specify which tags to strip either explicitly, or on save.
      */
     enum StripTags {
-      StripNone,  //<! Don't strip any tags
-      StripOthers //<! Strip all tags not explicitly referenced in method call
+      StripNone,  ///< Don't strip any tags
+      StripOthers ///< Strip all tags not explicitly referenced in method call
     };
 
     /*!
      * Used to specify if when saving files, if values between different tag
      * types should be synchronized.
      */
     enum DuplicateTags {
-      Duplicate,     //<! Synchronize values between different tag types
-      DoNotDuplicate //<! Do not synchronize values between different tag types
+      Duplicate,     ///< Synchronize values between different tag types
+      DoNotDuplicate ///< Do not synchronize values between different tag types
```

Comment:
`//!<` would be more consistent with the doxygen comment style used in the rest of the code.